### PR TITLE
feat(Cache): add `setValue` combinator.

### PR DESCRIPTION
**Pull request by patroza**

### CODE REVIEW

File: packages/cache/src/Cache/index.ts [hunk 1]
```diff
@@ -465,6 +419,18 @@ export function size<Key, Error, Value>(self: Cache<Key, Error, Value>): T.UIO<n
   return self.size
 }
 
+/**
+ * Sets the value in the cache.
+ */
+export function setValue_<Key, Error, Value>(
```
Comment (IMax153) [inline packages/cache/src/Cache/index.ts:425]: We should also add the data last combinator, i.e.

```typescript
export function setValue<Key, Value>(key: Key, value: Value) {
  return <Error>(self: Cache<Key, Error, Value>): T.UIO<void> => setValue_(self, key, value)
}
```

**Review by IMax153 — CHANGES_REQUESTED** [review]

**Comment by IMax153** [comment]
Also looks like a similar issue with `prettier` formatting in CI-CD. Not sure why your VSCode isn't auto-formatting 🤔

File: packages/cache/src/Cache/index.ts (hunk 1 above)
Comment (patroza) [reply packages/cache/src/Cache/index.ts:424]: Alrighty

**Comment by patroza** [comment]
@IMax153 ready for the next review.
tests and data-last added.

**Comment by patroza** [comment]
> Also looks like a similar issue with `prettier` formatting in CI-CD. Not sure why your VSCode isn't auto-formatting 🤔

apparently im used to having my formatter settings in the repository `.vscode/settings.json`.
I made em global user settings now ;-)

**Review by IMax153 — APPROVED** [review]
LGTM 👍 - thanks for your work on this.

**Comment by IMax153** [comment]
> apparently im used to having my formatter settings in the repository `.vscode/settings.json`

This is a good point - I'll add to the `.vscode/settings.json` 👍

**Pull request merged**